packages/ui/src/forum/modals/CreateThreadModal/CreateThreadDetailsModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react'
+
+import { ButtonPrimary, ButtonsGroup } from '@/common/components/buttons'
+import { CKEditor } from '@/common/components/CKEditor'
+import { InputComponent, InputText } from '@/common/components/forms'
+import { Arrow } from '@/common/components/icons'
+import { Modal, ModalBody, ModalFooter, ModalHeader } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TextMedium } from '@/common/components/typography'
+import { useModal } from '@/common/hooks/useModal'
+
+import { GeneralFormContext } from './machine'
+
+interface Props {
+  context: GeneralFormContext
+  setTopic: (t: string) => void
+  setDescription: (d: string) => void
+  onSubmit: () => void
+}
+
+export const CreateThreadDetailsModal = ({
+  context: { topic, description },
+  setTopic,
+  setDescription,
+  onSubmit,
+}: Props) => {
+  const fieldMissing = useMemo(() => !(topic && description), [topic, description])
```

Comment:
I'm really acustomed to `boolean` variables starting with `is`, `has`, `are`, etc. It is then easier to parse.

So here it would be either, standard `isValid` -> will show that this is some sort of "validation" as well.